02-Logging/02-Writing-to-a-File.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>Same comment as in the other notebook, this should link to something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>